### PR TITLE
Add docker image for ebpf library build

### PR DESCRIPTION
**Pull request by paskal**
Part of #5 resolution. This PR adds a docker image which can be used to bbuild ebpf library.

### CODE REVIEW

**Comment by paskal** [comment]
CI failure should be fixed in GitHub Actions, [this is the fix](https://github.com/github-actions-x/commit/pull/18) for the action we use.

**Comment by thecrogers** [comment]
Yeah, ideally this should only need to pull the image. Instead of building every time. 

You might be able to do a check if image exist, if not build and push.

**Comment by paskal** [comment]
@thecrogers please review once more, I reduced the scope of the PR to make it less controversial.

File: pkg/ebpf/Makefile [hunk 1]
```diff
@@ -3,13 +3,13 @@ KERNEL_RELEASE ?= $(shell uname -r)
...
 
 INCLUDES =  -I $(INCLUDE_DIR)
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/generated
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include/generated/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/asm/asm.h
+INCLUDES += -I /usr/src/kernels/$(KERNEL_RELEASE)/arch/x86/include
```
Comment (thecrogers) [inline pkg/ebpf/Makefile:6]: Did you test this in a non docket context? It seems unlikely the path would change this much.

Comment (nloginov) [reply pkg/ebpf/Makefile:6]: `/lib/modules/$(KERNEL_RELEASE)/build/arch` is a symlink to `/usr/src/kernels/$(KERNEL_RELEASE)/`

Comment (thecrogers) [reply pkg/ebpf/Makefile:6]: Ah yeah I remember reading that here recently. Thanks for the clarification.

**Review by thecrogers — APPROVED** [review]
LGTM.

File: pkg/ebpf/Makefile (hunk 1 above)
Comment (paskal) [reply pkg/ebpf/Makefile:6]: I did test it outside of docker, works fine.

**Review by nloginov — APPROVED** [review]

**Pull request merged**